task_6_8.py

```diff
@@ -0,0 +1,24 @@
+import os
+import urllib.request
+
+
+def get_text_from_url(url_text):
+    response = urllib.request.urlopen(url_text)
+    return response.read().decode('utf-8')
+
+
+def extract_chapters(text_book, o_file):
+    output_dir = 'data'
+    if not os.path.exists(output_dir):  # make decorator
+        os.makedirs(output_dir)
+
+    output_path = os.path.join(output_dir, o_file)
```

Comment:
you could skip everything related to dir creation. just put the file along.
OR
you could create a separate function for dir creation